pywinauto/base_types.py

```diff
@@ -115,3 +143,96 @@ def __getitem__(self, key):
             return self.y
         else:
             raise IndexError("Illegal index")
+
+
+# ====================================================================
+class RectExtMixin(object):
+
+    """Wrap the RECT structure and add extra functionality"""
+
+    # To be initiated by OS-specific types
+    _RECT = None
+    _POINT = None
+
+    # ----------------------------------------------------------------
+    def __init__(self, otherRect_or_left=0, top=0, right=0, bottom=0):
+        """Provide a constructor for _RECT structures
+
+        A _RECT can be constructed by:
+        - Another RECT (each value will be copied)
+        - Values for left, top, right and bottom
+
+        e.g. my_rect = _RECT(otherRect)
+        or   my_rect = _RECT(10, 20, 34, 100)
+        """
+        if isinstance(otherRect_or_left, self._RECT):
```

Comment:
Since the second argument should be a type, we have to initialize this static member as `_RECT = type(None)` (`types.NoneType`  works in Python 2.7 only).